python/pyarrow/tests/conftest.py

```diff
@@ -99,6 +100,7 @@
     'slow': False,
     'snappy': Codec.is_available('snappy'),
     'tensorflow': False,
+    'udf': False,
```

Comment:
If we're using the dataset mark why do we need this here?

Reply:
Ah yes, we should remove this. Good catch.